pkg/network/ebpf/c/protocols/kafka/kafka-parsing.h

```diff
@@ -526,15 +839,150 @@ static __always_inline enum parse_result kafka_continue_parse_response_loop(kafk
     return RET_LOOP_END;
 }
 
-static __always_inline enum parse_result kafka_continue_parse_response(kafka_info_t *kafka,
+static __always_inline void kafka_call_response_parser(void *ctx, conn_tuple_t *tup, pktbuf_t pkt, kafka_response_state state, u32 api_version)
+{
+    enum parser_level level = parser_state_to_level(state);
+    u32 index;
```

Comment:
```suggestion
    u32 index = 0;
```

Reply:
I see, can you please add a comment in the code clarifying this point?

Reply:
Done.